public/v4/apps/teslamate.yml

```diff
@@ -27,14 +27,15 @@ services:
         depends_on:
             - $$cap_appname-db
     $$cap_appname-mqtt:
-        caproverExtra:
-            notExposeAsWebApp: 'true'
-        image: eclipse-mosquitto:2
-        restart: always
-        command: mosquitto -c /mosquitto-no-auth.conf
         volumes:
             - $$cap_appname-mosquitto-conf:/mosquitto/config
             - $$cap_appname-mosquitto-data:/mosquitto/data
+        restart: always
+        caproverExtra:
+            dockerfileLines:
+                - FROM eclipse-mosquitto:2
+                - CMD exec mosquitto -c /mosquitto-no-auth.conf
```

Comment:
Thanks for the update!

Just to confirm, `exec` here is intentional and tested? Typically it's used to replace the shell with the program - and it's not usually used in Dockerfiles.

Reply:
Hey there, `exec` here is tested, and can confirm the config `mosquitto-no-auth.conf` is loaded (as seen in container log):
```log
1714964682: Opening ipv4 listen socket on port 1883.,1714964682: mosquitto version 2.0.18 starting,
1714964682: Opening ipv6 listen socket on port 1883.,
1714964682: mosquitto version 2.0.18 running,
1714964682: Config loaded from /mosquitto-no-auth.conf.
```

it both yield the same result, with or without `exec` after `docker inspect`:
```json
"Cmd": [
    "/bin/sh",
    "-c",
    "mosquitto -c /mosquitto-no-auth.conf"
]
```

It was added due to referencing the example in `redis-server.yml`, i will make another commit shortly to improve that tho.

thanks!